### ISSUE TITLE
How realistic is it to earn $3000-5000 per month on a cryptominer?

### ISSUE THREAD

**Issue by undercryptominer** [UnamSanctam/UnamWebPanel#423]
Good afternoon, ladies and gentlemen who have experience working with silent cryptominer. I would like to ask how much money is actually earned per month in general? Is it possible to earn at least $ 300, let alone several thousand? I'm just starting to work and that's why it's very interesting. Thank you for your answers!

**Comment by FrazeDev** [comment]
Yes its possible but you would need over then 5000 bots if iam not wrong but if you want to make more than 5 usd don't mine BTC mine shit coins and have atleast 1000-2000bots

**Comment by undercryptominer** [comment]
> Да, это возможно, но вам понадобится более 5000 ботов, если я не ошибаюсь. Но если вы хотите заработать больше 5 долларов, не майните BTC, майните дерьмовые монеты и заведите как минимум 1000-2000 ботов.

Thank you for your reply! Please tell me, is it possible to recruit 5,000 bots in a few months of hard work?

**Comment by undercryptominer** [comment]
> Yes its possible but you would need over then 5000 bots if iam not wrong but if you want to make more than 5 usd don't mine BTC mine shit coins and have atleast 1000-2000bots

Thank you for your reply! Please tell me, is it possible to recruit 5,000 bots in a few months of hard work?

**Comment by FrazeDev** [comment]
> > Yes its possible but you would need over then 5000 bots if iam not wrong but if you want to make more than 5 usd don't mine BTC mine shit coins and have atleast 1000-2000bots
> 
> Thank you for your reply! Please tell me, is it possible to recruit 5,000 bots in a few months of hard work?

Yes its always possible but you just have to have more than 1 spreading method or you can try making your own

**Comment by FrazeDev** [comment]
> > > > Yes its possible but you would need over then 5000 bots if iam not wrong but if you want to make more than 5 usd don't mine BTC mine shit coins and have atleast 1000-2000bots
> > > 
> > > 
> > > Thank you for your reply! Please tell me, is it possible to recruit 5,000 bots in a few months of hard work?
> > 
> > Yes its always possible but you just have to have more than 1 spreading method or you can try making your own
> 
> Hi, I'm sorry for the offtop, but I'm also a beginner and I can't run the miner, so I'm asking for advice. Both ETC and XMR miner show an error (as seen from the screenshot), and XMR couldn't even connect to the pool. It is important to note that I used the full version of the miner settings. What do you think might be the problem? Outdated processor and graphics card? Or something else? I will be grateful for any help
> ![1](https://github.com/user-attachments/assets/8b157d93-4b62-43a6-a3db-b51e46128d8f)
> ![2](https://github.com/user-attachments/assets/4a37815d-6d35-4a43-a073-8c0055e77121)
> 


Iam not sure if you can even mine eth cash but the gpus is to weak to mine and for xmr try changing the pool as it can be blocked in some country's or by the firewall

**Comment by masterjek** [comment]
> > > > Да, это возможно, но вам понадобится более 5000 ботов, если я не ошибаюсь. Но если вы хотите заработать больше 5 долларов, не майните BTC, майните дерьмовые монеты и заведите как минимум 1000-2000 ботов.
> > > 
> > > 
> > > Спасибо за ответ! Скажите, пожалуйста, возможно ли набрать 5000 ботов за несколько месяцев упорной работы?
> > 
> > 
> > Да, это всегда возможно, но вам просто нужно иметь более одного метода распределения или вы можете попробовать сделать свой собственный.
> 
> Привет, извините за оффтоп, но я тоже новичок и не могу запустить майнер, поэтому прошу совета. И ETC и XMR майнер выдают ошибку (как видно на скриншоте), а XMR даже не смог подключиться к пулу. Важно отметить, что я использовал полную версию настроек майнера. В чем может быть проблема, как вы думаете? Устаревший процессор и видеокарта? Или что-то еще? Буду благодарен за любую помощь ![1](https://private-user-images.githubusercontent.com/183024860/372903711-8b157d93-4b62-43a6-a3db-b51e46128d8f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MDY0NDEsIm5iZiI6MTcyNzkwNjE0MSwicGF0aCI6Ii8xODMwMjQ4NjAvMzcyOTAzNzExLThiMTU3ZDkzLTRiNjItNDNhNi1hM2RiLWI1MWU0NjEyOGQ4Zi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQyMTU1NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmI2ODk4MTkzMDA5NjhkM2ZhNjhhYzQ4YmU4ZGJjNTdiYmEwYWM5ZmMyNzZiMmEzZjgzMDczZjFlZWQ2NGFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7R0_-yr8mMH6zViOIRb_jKBLfsm1luHJCFS6PY_Nqpo) ![2](https://private-user-images.githubusercontent.com/183024860/372903718-4a37815d-6d35-4a43-a073-8c0055e77121.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc5MDY0NDEsIm5iZiI6MTcyNzkwNjE0MSwicGF0aCI6Ii8xODMwMjQ4NjAvMzcyOTAzNzE4LTRhMzc4MTVkLTZkMzUtNGE0My1hMDczLThjMDA1NWU3NzEyMS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwMlQyMTU1NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMzQ3MjAzNmM5YTYwZmI2ZjExM2E5ZjAzMzY1MTlkNDFmN2NlNDY4Y2I4YmM3M2UzOGY3OWVjMzM1NDJjZjFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.2E_mNOxI0_J_dswDsrq2vLZ5h7ElIrYcxEWjGwisFk0)

1. Your video card Nvidia GeForce GT 730 - Doesn't have enough video memory to mine Etc - That's why it gives an error.
2. xmr miner error usually pool connection error occurs because the pool blocks access from certain countries.

**Comment by error0x1337** [comment]
There is no limit earnings :) u can earn even 100k a month. it only depents on your bots.

**Comment by error0x1337** [comment]
> There is no limit earnings :) u can earn even 100k a month. it only depends on your bots.
